src/renderer/store/modules/settings.js

```diff
@@ -296,6 +296,8 @@ const state = {
   screenshotFolderPath: '',
   screenshotFilenamePattern: '%Y%M%D-%H%N%S',
   settingsSectionSortEnabled: false,
+  showAddedChannels: true,
+  showAddedKeywords: true,
```

Comment:
1. These should be moved to below `forbiddenTitles` (coz they are related)
2. I think maybe using `showAddedChannelsHidden` & `showAddedForbiddenTitles` would be better to clearly show that they are related.